dcap_quoteverify/sys/build.rs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2022 The MobileCoin Foundation
+//! Builds the FFI function bindings for dcap quoteverify library of the Intel
+//! SGX SDK
+
+use cargo_emit::rustc_link_lib;
+
+const DCAP_QL_FUNCTIONS: &[&str] = &[
+    "sgx_qv_free_qve_identity",
+    "sgx_qv_get_quote_supplemental_data_size",
+    "sgx_qv_get_qve_identity",
+    "sgx_qvl_get_quote_supplemental_data_size",
+    "sgx_qvl_get_quote_supplemental_data_version",
+    "sgx_qvl_verify_quote",
+    "sgx_qv_set_enclave_load_policy",
+    "sgx_qv_set_path",
+    "sgx_qv_set_qpl_path",
```

Comment:
`sgx_qv_set_qpl_path` and `sgx_qv_set_qve_path` are both defined in `sgx_dcap_pcs_com.h`, which isn't listed in `wrapper.h`. That header looks like a public header so I think it should be added to `wrapper.h`.

Separately, that header also defines a number of other functions and several function pointer types which should probably be bound as well.

Reply:
removed `sgx_qv_set_qpl_path`.  that is accessed internally via `sgx_qv_set_path`.  The `sgx_dcap_pcs_com.h` looks to defining functions for the `sgx_dcap_quoteprov` library.  There is one provided with the DCAP build, or one can choose to provide their own implementation, but these are all accessed via the DCAP c libraries and shouldn't need to be exposed to the rust side of things at all.